### PR TITLE
feat(theme-default): redesign custom container

### PR DESCRIPTION
**Pull request by CikiMomogi**
Result:
- Before: https://cikimomogi-grammy.github.io/vp-main/result.html
- After: https://cikimomogi-grammy.github.io/vp-1131/result.html

### CODE REVIEW

**Comment by Mister-Hope** [comment]
I personally don't think that we do need to change details container color in other blocks.

You may sperate this in several PR to have a better discussion about improving colors.

E.g.: I do think a fix for table 2n rows is needed.

**Comment by CikiMomogi** [comment]
Okay i will split them into another PR

**Comment by meteorlxy** [comment]
You'd better provide Before and After screenshots for UI changes.

**Comment by CikiMomogi** [comment]
Result:
- Before: https://cikimomogi-grammy.github.io/vp-main/result.html
- After: https://cikimomogi-grammy.github.io/vp-1131/result.html

**Comment by CikiMomogi** [comment]
I also plan to modify the color palettes for `table` and `details container`. What do you think? 

**The background color of `Code` is too dark. Adding some adjustment.**
_Table before:_
table-before
_Table after:_
table-after

**The `details` container inside the danger container. Make the background color match the color of the parent container.**
_Details Before:_
details-before
_Details after:_
details-after

**Comment by CikiMomogi** [comment]
Update:
- Before: https://cikimomogi-grammy.github.io/vp-main/result.html
- After: https://cikimomogi-grammy.github.io/vp-1131/result.html

File: ecosystem/theme-default/src/client/styles/code.scss [hunk 1]
```diff
@@ -264,3 +264,13 @@ div[class*='language-'] {
...
+
+// ===============================
+
+tr {
+  &:nth-child(2n) {
+    code {
+      background-color: var(--c-bg-dark);
+    }
+  }
+}
```
Comment (meteorlxy) [inline ecosystem/theme-default/src/client/styles/code.scss:276]: Maybe we should put them to the table styles file.

File: ecosystem/theme-default/src/client/styles/vars.scss [hunk 2]
```diff
@@ -29,16 +31,28 @@
   --c-tip-title: var(--c-text);
   --c-tip-text: var(--c-text);
   --c-tip-text-accent: var(--c-text-accent);
-  --c-warning: #e7c000;
+  --c-warning: #ffc310;
```
Comment (meteorlxy) [inline ecosystem/theme-default/src/client/styles/vars.scss:34]: There vars are also used by badge component. Could you help to check the styles of them?

**Review by meteorlxy — CHANGES_REQUESTED** [review]
Most changes look good. The badge component is also affected so plz add them to the before / after preview, too.

File: ecosystem/theme-default/src/client/styles/code.scss (hunk 1 above)
Comment (CikiMomogi) [reply ecosystem/theme-default/src/client/styles/code.scss]: Done at commit https://github.com/vuepress/vuepress-next/pull/1131/commits/9045a4f8303eec5b48fdbb6802ef0d8ba7042dc0

File: ecosystem/theme-default/src/client/styles/vars.scss (hunk 2 above)
Comment (CikiMomogi) [reply ecosystem/theme-default/src/client/styles/vars.scss:34]: Done at commit https://github.com/vuepress/vuepress-next/pull/1131/commits/1bb9e38135802998805e919bd3a34504d6519339

**Comment by CikiMomogi** [comment]
Update:
- Before: https://cikimomogi-grammy.github.io/vp-main/result.html
- After: https://cikimomogi-grammy.github.io/vp-1131/result.html

**Review by meteorlxy — APPROVED** [review]

**Pull request merged**